src/main/java/org/apache/commons/io/output/FileWriterWithEncoding.java

```diff
@@ -160,7 +209,11 @@ public FileWriterWithEncoding(final File file, final String charsetName) throws
      * @throws IOException in case of an I/O error.
      */
     public FileWriterWithEncoding(final File file, final String charsetName, final boolean append) throws IOException {
-        this.out = initWriter(file, charsetName, append);
+        super(NULL_WRITER);
+
+        super.out = initWriter(file, charsetName, append);
```

Comment:
Ah, I think now I understand this pull request.

@wodencafe I think we pass the `NULL_WRITER` to the parent `ProxyWriter` as a way to give it a valid object. But in reality, that `NULL_WRITER` is then replaced here when we call `super.out = ....`.

While I guess that would work, but not sure if modifying setting `super.out` twice (once after the `super(NULL_WRITER)`, then here :point_up: ) would be the best approach.